src/onchain/channel_closes.rs

```diff
@@ -239,7 +239,7 @@ impl ChannelClose {
 
         let lsp_fees = self
             .support
-            .calculate_lsp_fee_for_amount(send_amount_sats)
+            .calculate_lsp_fee_for_amount(send_amount_sats, None)
```

Comment:
This logic needs to be fixed anyway (we should use lsp fee params provided as input -  let's fix in another PR), but I would use here the new swap version of the method.

Reply:
I'm gonna create a second PR :+1:

Reply:
Temporary fix: 9854933355983b8f7fc790547759b225ce1dc4eb